sources/assembly-data/ATTR_assembly.types.yaml

```diff
@@ -116,35 +146,46 @@ attributes:
       count: 10
       scale: log10
   protein_count:
+    description: Protein count
+    long_description: Number of proteins in an assembly derived from annotation
     display_group: annotation
     display_level: 2
     display_name: Protein count
     constraint:
       min: 1
     type: integer
   pseudogene_count:
+    description: Pseudogene count
+    long_description: Number of pseudogenes in an assembly derived from annotation
     display_group: annotation
     display_level: 2
     display_name: Pseudogene count
     constraint:
       min: 1
     type: integer
   noncoding_gene_count:
+    description: Non-coding gene count
+    long_description: Number of noncoding genes in an organism's genome assembly
     display_group: annotation
     display_level: 2
     display_name: Non-coding gene count
     constraint:
       min: 1
     type: integer
   sample_sex:
+    description: Sample sex
     display_level: 2
     display_name: Sample sex
     type: keyword
   isolate:
+    description: Isolate
     display_level: 2
     display_name: Isolate
     type: keyword
   assembly_level:
+    description: Highest level of an assembly submission
+    long_description: The highest assembly level for any object in the assembly. The assembly level is a proxy for existence of an assembly of the whole nuclear genome of a taxon. For a taxon, having a value for the assembly_level attribute can be considered the same as having a genome assembly
```

Comment:
ordered from lowest to highest, the assembly level can be "contig", "scaffold"...

proxy for _the_ existence